src/theme/CodeBlock/index.js

```diff
@@ -0,0 +1,47 @@
+import React, {useState, useRef, useCallback, useEffect} from 'react';
```

Comment:
@lio-p this is where our code block will go. Pay attention to lazy-loading - @Blargian found this to important for google site metricsw